### PR TITLE
docs: add warnings related to API v2 and v2.4 in instance view documentation

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: doc/instanceVisualization.md [hunk 1]
```diff
@@ -22,13 +22,21 @@ To create this new App Registration:
   in the Azure Portal
 - click on the _“+ New registration”_ button
   - choose the name of your Function App (you can use the pattern
-    `<your_solution_name><staging_env>ScenarioDownloadFunctionApp` (e.g. _“BreweryDevScenarioDownloadFunctionApp”_)
+    `<your_solution_name><staging_env>ScenarioDownloadFunctionApp`, e.g. _“BreweryDevScenarioDownloadFunctionApp”_)
```
Comment (Guismus) [inline doc/instanceVisualization.md:25]: maybe missing ( symbol ?

File: doc/instanceVisualization.md [hunk 2]
```diff
@@ -425,11 +443,19 @@ Check the function **URL** and **secret** in the front-end configuration file.
 
 Check the error log in your Function App to get more information on the error.
 
-If the error message is _“Access is denied”_:
+If the error message is **_“Access is denied”_**:
 
 - check that your app registration has the permissions `Dataset.Reader`, `Scenario.Reader` and `Workspace.Reader`
 - check that these permissions have been granted by an administrator
 
+If the error message complains about a **scenario not found**:
```
Comment (Guismus) [inline doc/instanceVisualization.md:451]: **_scenario not found_**: seems better with other if messages

File: doc/instanceVisualization.md (hunk 1 above)
Comment (csm-thu) [reply doc/instanceVisualization.md:25]: Actually, no, the point of this change is to remove an extra open parenthesis :slightly_smiling_face:

**Review by ardvi — APPROVED** [review]
LGTM :+1:

File: doc/instanceVisualization.md (hunk 2 above)
Comment (csm-thu) [reply doc/instanceVisualization.md:451]: The problem is that unlike the two other error messages, this "scenario not found" is not an exact quote, the error message tends to change based on the API version used. That's why I preferred a more vague sentence with "complains about ..." :smile:

**Pull request merged**